test/result.nix

```diff
@@ -8,12 +8,12 @@ let
   self = packages // (overrides self packages);
 
   packages = with beamPackages; with self; {
-    cowboy = buildErlangMk rec {
+    cowboy = buildRebar3 rec {
```

Comment:
In the past, I've had trouble installing cowboy with buildRebar3. That is why there is a manual workaround to use buildErlangMk.

Reply:
Hmm, I made this change because I ran `mix test` and it failed because it was generating cowboy with `buildRebar3`.

Reply:
Ok this was a bug I introduced that broke the specific workarounds. I was passing atoms instead of strings, so none of the workarounds were matching. Fixing.